app/javascript/components/project-submissions/containers/project-submissions-container.js

```diff
@@ -85,6 +85,32 @@ const ProjectSubmissions = (props) => {
     }
   };
 
+  const toggleLikeSubmission = async (submission) => {
+    const response = await axios.post(
+      `/project_submissions/${submission.id}/likes`,
+      {
+        submission_id: submission.id,
+        is_liked_by_current_user: submission.is_liked_by_current_user
+      }
+    );
+
+    if (response.status === 200) {
+      const updatedSubmission = response.data;
+
+      setSubmissions(prevSubmissions => {
+        const newSubmissions = prevSubmissions.map((sub) => {
```

Comment:
Nitpick, can we change `sub` to `submissions` please? abbreviations are the devil 😆